modules/carto/src/api/layer-map.ts

```diff
@@ -153,7 +158,12 @@ export function getLayer(
   > = {
     point: {
       Layer: GeoJsonLayer,
-      propMap: {visConfig: {outline: 'stroked'}}
+      propMap: {
+        columns: {
+          altitude: x => ({parameters: {depthTest: x?.fieldIdx > -1}})
```

Comment:
It looks like we not only _disable_ depthTest in most cases, but enable for some very specific ones. Why not adding test for this scenario?

Second, i can only generate maps with `columns.altitude` with type `string`:
```
{
    "lat": "latitude",
    "lng": "longitude",
    "altitude": "year"
}
```

Please check if this condition is correct.

Reply:
Changed to test for "null" and I've updated the tests to explicitly test for the `depthTest: true` case